packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageSearchBar.test.js

```diff
@@ -61,16 +61,16 @@ describe('<TracePageSearchBar>', () => {
         expect(button.hasClass('is-disabled')).toBe(false);
         expect(button.prop('disabled')).toBe(false);
       });
-      expect(wrapper.find('Button[icon="up"]').prop('onClick')).toBe(defaultProps.prevResult);
-      expect(wrapper.find('Button[icon="down"]').prop('onClick')).toBe(defaultProps.nextResult);
-      expect(wrapper.find('Button[icon="close"]').prop('onClick')).toBe(defaultProps.clearSearch);
+      expect(wrapper.find('[data-testid="UpOutlined"]').prop('onClick')).toBe(defaultProps.prevResult);
+      expect(wrapper.find('[data-testid="DownOutlined"]').prop('onClick')).toBe(defaultProps.nextResult);
+      expect(wrapper.find('[data-testid="CloseOutlined"]').prop('onClick')).toBe(defaultProps.clearSearch);
     });
 
     it('hides navigation buttons when not navigable', () => {
       wrapper.setProps({ navigable: false });
       const button = wrapper.find('Button');
       expect(button.length).toBe(1);
-      expect(button.prop('icon')).toBe('close');
+      expect(button.getElement(CloseOutlined)).toBeDefined();
```

Comment:
can you not look it up by data-testid to avoid explicit dependency on ant icons?

Reply:
Yes, maybe it could be done that way too. I will try and update.